tests/cpp/utils.h

```diff
@@ -169,6 +169,31 @@ class PredicatedChecker : public kir::IrVisitor {
     return isPredicated(tv->name(), kernel);
   }
 
+  static bool isPredicatedByIfThenElse(
+      StmtNameType tv_name,
+      kir::Kernel* kernel) {
+    PredicatedChecker checker(tv_name, kernel->topLevelExprs());
+    return checker.predicated_ite_;
+  }
+
+  static bool isSharedMemoryPredicatedByIfThenElse(kir::Kernel* kernel) {
```

Comment:
Should we update the name for the latest change?

Reply:
changed to `isCpAsyncMmaPredicatedByIfThenElse` and added comments.